doc/source/user_guide/io.rst

```diff
@@ -365,17 +360,14 @@ columns:
 .. ipython:: python
 
     import numpy as np
-    data = ('a,b,c,d\n'
-            '1,2,3,4\n'
-            '5,6,7,8\n'
-            '9,10,11')
+
+    data = "a,b,c,d\n1,2,3,4\n5,6,7,8\n9,10,11"
     print(data)
 
     df = pd.read_csv(StringIO(data), dtype=object)
     df
-    df['a'][0]
-    df = pd.read_csv(StringIO(data),
-                     dtype={'b': object, 'c': np.float64, 'd': 'Int64'})
+    df["a"][0]
+    df = pd.read_csv(StringIO(data), dtype={"b": object, "c": np.float64, "d": "Int64"})
```

Comment:
Also, we should not use black's default line length limit of 88, because that will not fit in the code block width of the doc page. For example, this line is 85, and only fits exactly. So anything bigger will cause overflow/scrollbar.

Reply:
Yeah I was wondering about this afterwards, will work on fixing these